controller/index.js

```diff
@@ -1,9 +1,8 @@
 var https = require('https')
 var Sm = require('sphericalmercator')
 var merc = new Sm({size: 256})
-var crypto = require('crypto')
-var _ = require('lodash')
 var fs = require('fs')
+var Utils = require('../models/utils.js')
```

Comment:
I know this is an artifact of @chelm's refactor but it's a little weird to have a `utils` module inside of `models`. Would probably be better if it were under `lib` in the root since it's being used by the controller too.